python/modules/python_connection.py

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/python
+import importlib.util
+import logging
+
+logger = logging.getLogger("IPIQA.python_connection")
+
+def execute_python_script(param_dict, python_script_path, function_name):
+    p_source = importlib.util.spec_from_file_location("external_module", python_script_path)
```

Comment:
Add some comment this function. How script should look like ? Do you load python script and then execute functions that are there ? 
What if python script is bigger than one file ?